wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/ru.md

```diff
@@ -0,0 +1,92 @@
+
+## Базовые требования
+
+От моддеров, желающих стать номинаторами, ожидается следующее:
+
+- Активный моддинг в течение минимум 3 месяцев перед подачей заявки;
+- Как минимум 150 или 200 набранных [кудосу](/wiki/Modding/Kudosu) (зависит от режима игры);
+- Ссылка на 2–3 мода, демонстрирующих навыки кандидата в роли номинатора.
+
+### Качество моддинга
+
+Моды, приводимые в качестве примеров, должны отражать навык анализа и улучшения карт на уровне номинатора.
+
+Ниже перечислены качества, которыми должны обладать номинаторы. Вашу заявку могут отклонить, если в перечисленных в ней модах не видно хотя бы одной из этих черт.
+
+- **Моддинг всех уровней сложности;**
+- **Умение замечать неранкабельные проблемы, включая те, которые нельзя обнаружить автоматически (например, неправильный тайминг или метаданные);**
+- **Сопоставление разных участков карты как аргументация своих замечаний;**
+- **Замечания, направленные на разные стороны карты, например, на ритмику, спейсинг, движение, интенсивность, контраст и последовательность;**
+- **Умение находить как конкретные недочёты, так и проблемы, присущие всей карте целиком;**
+- **Понимание намерений маппера при поиске проблем или возможных улучшений.**
+
+### Процесс отбора заявкок
+
+- Поданные заявки проверяют члены NAT соответствующего игрового режима. Они обращают внимание на следующие стороны:
+  - Общее поведение: тон общения и подача модов, а также другие пункты из [соответствующего свода правил](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping), включая поведение в чате и на форумах в принципе.
+  - Навыки моддинга: знание [критериев ранкинга](/wiki/Ranking_Criteria) — как общих, так и для отдельных режимов. Также учитываются дополнительные умения (поиск метаданных, работа с таймингом и т.д.)
+- После того, как все заявки рассмотрены, члены NAT обсуждают кандидатов и решают, кто из них готов стать номинатором.
+
+## Уровень моддинга
+
+Для того, чтобы стать номинатором в режиме osu!, необходимо иметь определённый уровень активности. Это требование пока **не** применяется к прочим режимам (osu!taiko, osu!catch и osu!mania).
+
+### Какие моды учитываются
+
+- После авторизации на веб-сайте для NAT вы увидите число кудосу, полученных за последние 3 месяца за мод-посты.
+- При подсчёте кудосу учитываются только посты за последние 3 месяца в обсуждениях карт, в которых хотя бы одно предложение или проблема были одобрены.
+- Режим карты, которую вы модили, не учитывается, поскольку измеряется ваша активность в моддинг-сообществе в целом.
+- Дата выдачи кудосу **имеет значение**, так как при подсчёте используется ваша история моддинга из профиля.
+
+### Требования по баллам
+
+- Требования зависят от того, во сколько баллов — `ModScore` — оценивается заявка:
+
+![Формула ModScore](/wiki/shared/Modscore_new_wiki.png "Формула ModScore")
+
+`m` — сколько минимум различных карт необходимо моддить в месяц;\
```

Comment:
```suggestion
`m` — сколько минимум различных карт необходимо моддить в месяц;
```

Reply:
it's a line break required for two lines to stack -- see https://osu.ppy.sh/wiki/en/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator#how-the-required-minimum-score-is-calculated